forum/src/main/java/unipd/se18/ocrcamera/forum/ForumLogin.java

```diff
@@ -0,0 +1,182 @@
+package unipd.se18.ocrcamera.forum;
+
+import android.arch.lifecycle.Observer;
+import android.arch.lifecycle.ViewModelProvider;
+import android.arch.lifecycle.ViewModelProviders;
+import android.content.Context;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import unipd.se18.ocrcamera.forum.viewmodels.Login_VM;
+
+
+/**
+ * A fragment where a user can login to the forum service
+ *
+ * @author Leonardo Rossi (g2), Alberto Valente (g2), Taulant Bullaku (g2)
+ */
+public class ForumLogin extends Fragment {
+
+    /**
+     * ***************************
+     * **   GLOBAL VARIABLES    **
+     * ***************************
+     */
+
+    private OnFragmentInteractionListener mListener;
```

Comment:
This variable is never initialized and use, i think u can delete it

Reply:
You're right: I think I forgot to delete it after the recent refactoring, so I'll get it done.